doc/org/build/networking.md

```diff
@@ -0,0 +1,256 @@
+-   The `takeSoftOwnership` function allows clients to take ownership of unowned entities in such a way that only clients with the most recent information about that entity will be eligible as the new owner.
+
+For the most part, users of the networking systems do not need to understand these concepts. These are handled internally by the systems themselves. However, users do need to understand that ownership is not transactional or guaranteed. That is, ownership is not &ldquo;requested and then transferred&rdquo;, and just because one client claims ownership of an entity does not mean that other clients will respect that claim.
+
+Users can inspect the state the `Networked` or `Owned` components as needed in cases when their ownership claims matter. They may find themselves writing coroutines that looks like this:
+
+```typescript
+takeSoftOwnership(world, eid);
+yield sleep( 3000 ); // Wait a few seconds to see if we "win" ownership
+if (!hasComponent(world, Owned, eid)) return;
+```
+
+If this becomes a common and error-prone pattern, then we may introduce helper functions or additional semantics to cover these cases.
+
+
+<a id="org62cc95d"></a>
+
+# Creating Networked Entities
+
+Client code creates networked entities by calling `createNetworkedEntity`, passing:
+
+-   A `prefabName`, to indicate which prefab to initialize,
+-   An `initialData` struct, to know how to initialize the prefab.
+
+Prefabs must be registered in `prefabs` , a map from `PrefabName` to `PrefabDefinition`.
+
+`PrefabDefinition` s include `template` functions that take `InitialData` and return `EntityDef` s. `EntityDef` s are defined in `jsx`, using the `createElementEntity` transformer (not `React` &rsquo;s `createEntity` transformer).
+
+For example, the commonly used `"media"` prefab&rsquo;s `template` is:
+
+```typescript
+export function MediaPrefab(params: MediaLoaderParams): EntityDef {
+  return (
+    <entity
+      name="Interactable Media"
+      networked
+      networkedTransform
+      mediaLoader={params}
+      deletable
+      grabbable={{ cursor: true, hand: true }}
+      destroyAtExtremeDistance
+      floatyObject={{
+        flags: FLOATY_OBJECT_FLAGS.MODIFY_GRAVITY_ON_RELEASE,
+        releaseGravity: 0
+      }}
+      networkedFloatyObject={{
+        flags: FLOATY_OBJECT_FLAGS.MODIFY_GRAVITY_ON_RELEASE
+      }}
+      rigidbody={{ collisionGroup: COLLISION_LAYERS.INTERACTABLES, collisionMask: COLLISION_LAYERS.HANDS }}
+      physicsShape={{ halfExtents: [0.22, 0.14, 0.1] }} /* TODO Physics shapes*/
+      scale={[1, 1, 1]}
+    />
+  );
+}
+```
+
+When supplied with `MediaLoaderParams` as its `InitialData`, this prefab `template` creates an entity with several components, including a `Networked` and `NetworkedTransform` component.
+
+Calling `createNetworkedEntity(world, "media", mediaLoaderParams)` would cause the `networkSendSystem` to send a `CreateMessage` the next time it sends messages.
```

Comment:
Let me confirm my understanding. It is impossible to add or remove a certain component to an networked entity later, isn't it?